AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -242,6 +242,8 @@ hentaidude.com#@#.adsArea
 @@||coop-land.ru/*/js/ads.js
 @@||gohofstra.com/common/templates/dfp/dfp-
 @@||tags.news.com.au/prod/adblock/adblock.js
+turkanime.net###modallar
```

Comment:
It seems that this rule will also hide sign in/register popup.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/68760341-e71efd80-0611-11ea-95f4-b4327cc3ee2c.png)

</details>

I think that this rule should fixes anti-adblock:
```
turkanime.net#$#.pub_300x250.pub_300x250m.pub_728x90.text-ad.textAd.text_ad.text_ads.text-ads.text-ad-links { display:block!important; }
```
It seems that it's "FuckAdBlock", but if I'm not wrong, scriptlet doesn't work in this case, so I guess you can add this domain to these rules:
https://github.com/AdguardTeam/AdguardFilters/blob/4b7beb7fa0c7ab15492b833b71dcc3ed702e0cc8/AnnoyancesFilter/sections/antiadblock.txt#L147-L158